### PR TITLE
[MIG] Make mrp_bom_note installable in v9

### PR DESCRIPTION
**Pull request by oihane**
- mrp_bom_note

### CODE REVIEW

**Comment by pedrobaeza** [comment]
One module per PR, please.

**Comment by oihane** [comment]
Ok, changed, but what about the other two modules that one depends on the other?

File: mrp_bom_note/views/mrp_bom_view.xml [hunk 1]
```diff
@@ -2,18 +2,18 @@
 <openerp>
     <data>
```
Comment (max3903) [inline mrp_bom_note/views/mrp_bom_view.xml]: `<data>` can be removed.

Comment (oihane) [inline mrp_bom_note/views/mrp_bom_view.xml]: @max3903 is that something about v9? What in case it is `noupdate`?

File: mrp_bom_note/README.rst [hunk 2]
```diff
@@ -1,11 +1,18 @@
...
-===============
+==========================
+Notes in Bill of Materials
+==========================
 
 This module creates in BoM a new field to add notes
 
+Usage
+=====
+
```
Comment (max3903) [inline mrp_bom_note/README.rst]: * Go to Manufacturing > ...
* Edit an existing BOM and fill in the Notes field on tab X after field Y

File: mrp_bom_note/views/mrp_bom_view.xml (hunk 1 above)
Comment (max3903) [inline mrp_bom_note/views/mrp_bom_view.xml]: @oihane I don't know if `noupdate` works in `<odoo>`, but you don't have any here so it should work.

**Comment by oihane** [comment]
@pedrobaeza @max3903 there is some issue with coveralls, but module is ready for review

**Comment by max3903** [comment]
:+1: 

@oihane To answer your question about dependency, you need to migrate the dependencies first and once merged, you rebase to migrate your target module.

**Comment by pedrobaeza** [comment]
:+1: Merging to continue the rest of the modules migration

**Pull request merged**